src/interfaces/skylark/parsedObjects.ts

```diff
@@ -33,6 +33,7 @@ export interface ParsedSkylarkObjectContentObject {
   config: ParsedSkylarkObjectConfig;
   object: ParsedSkylarkObjectMetadata;
   position: number;
+  isNewObject?: boolean;
```

Comment:
don't think this should be here

Reply:
let's talk about alternatives 😅

Reply:
`AddedParsedSkylarkObjectContentObject` is better than custom but not sure